### PR TITLE
Backstage code review

### PR DESCRIPTION
**Pull request by armandocomellas1**
#### :heavy_check_mark: Checklist

- [x]  ChangeSets done:
## '/eleven-bags-flash.md' added extra parameter in the config settings (GkeClusterLocator.ts)
([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Before login BUG UI issue:

image

- [x] After login BUG UI fixed:

image
 
- [ ] All commits have a `Signed-off-by` line in the message. 
([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: .changeset/eleven-bags-flash.md
Comment (mondragonda) [inline .changeset/eleven-bags-flash.md:1]: The changeset is a group of modifications that are related to a feature or bug fix. So removing a console.log statement is not related to the necessary code to implement the feature or fix the bug. Is better to remove this changeset file.

File: plugins/kubernetes-backend/src/cluster-locator/GkeClusterLocator.ts [hunk 1]
```diff
@@ -121,7 +124,7 @@ export class GkeClusterLocator implements KubernetesClustersSupplier {
           // TODO filter out clusters which don't have name or endpoint
           name: r.name ?? 'unknown',
           url: `https://${r.endpoint ?? ''}`,
-          authMetadata: { [ANNOTATION_KUBERNETES_AUTH_PROVIDER]: 'google' },
+          authMetadata: { [ANNOTATION_KUBERNETES_AUTH_PROVIDER]: authProvider },
```
Comment (mondragonda) [inline plugins/kubernetes-backend/src/cluster-locator/GkeClusterLocator.ts:127]: The string authProvider means that authProvider is googleServiceAccount?

**Review by mondragonda — CHANGES_REQUESTED** [review]

File: plugins/kubernetes-backend/src/cluster-locator/GkeClusterLocator.ts (hunk 1 above)
Comment (armandocomellas1) [reply plugins/kubernetes-backend/src/cluster-locator/GkeClusterLocator.ts:127]: actually authProvider is variable, and it has by default 'google' value but when user set the config gke in the app-config.yaml and sets the authprovider: googleServiceAccount now that variable has it...

File: .changeset/eleven-bags-flash.md
Comment (armandocomellas1) [reply .changeset/eleven-bags-flash.md:1]: Agreed.

**Comment by mondragonda** [comment]
approved ✅

**Pull request merged**